src/views/Scenario/Scenario.js

```diff
@@ -276,94 +292,90 @@ const Scenario = (props) => {
-              <Grid container spacing={0} justifyContent="flex-end" className={classes.mainGrid}>
-                <Grid item>
-                  <PermissionsGate authorizedPermissions={[PERMISSIONS.canCreateScenario]}>
-                    <CreateScenarioButton
-                      solution={solution}
-                      workspaceId={workspaceId}
-                      createScenario={expandParametersAndCreateScenario}
-                      currentScenario={currentScenario}
-                      runTemplates={filteredRunTemplates}
-                      datasets={datasetList.data}
-                      scenarios={scenarioList.data}
-                      user={user}
-                      disabled={editMode}
-                      labels={createScenarioDialogLabels}
-                    />
-                  </PermissionsGate>
-                </Grid>
-              </Grid>
+          </Grid>
+          {currentScenario.data && (
+            <Grid item xs={4} className={classes.alignRight}>
+              {scenarioValidationArea}
             </Grid>
+          )}
+          <Grid item xs={12}>
+            <Card component={Paper} elevation={2}>
+              {currentScenario.data && (
+                <ScenarioParameters
+                  editMode={editMode}
+                  changeEditMode={setEditMode}
+                  addDatasetToStore={addDatasetToStore}
+                  updateAndLaunchScenario={updateAndLaunchScenario}
+                  launchScenario={launchScenario}
+                  accordionSummaryExpanded={accordionSummaryExpanded}
+                  onChangeAccordionSummaryExpanded={setAccordionSummaryExpanded}
+                  workspaceId={workspaceId}
+                  solution={solution.data}
+                  datasets={datasetList.data}
+                  currentScenario={currentScenario}
+                  scenarioId={currentScenario.data.id}
+                  scenarioList={scenarioList.data}
+                  userRoles={user.roles}
+                />
+              )}
+            </Card>
           </Grid>
           <Grid item xs={12}>
-            {currentScenario.data && (
-              <ScenarioParameters
-                editMode={editMode}
-                changeEditMode={setEditMode}
-                addDatasetToStore={addDatasetToStore}
-                updateAndLaunchScenario={updateAndLaunchScenario}
-                launchScenario={launchScenario}
-                accordionSummaryExpanded={accordionSummaryExpanded}
-                onChangeAccordionSummaryExpanded={setAccordionSummaryExpanded}
-                workspaceId={workspaceId}
-                solution={solution.data}
-                datasets={datasetList.data}
-                currentScenario={currentScenario}
-                scenarioId={currentScenario.data.id}
-                scenarioList={scenarioList.data}
-                userRoles={user.roles}
+            <Card component={Paper} elevation={2}>
```

Comment:
We lost paddings on scenario placeholders by removing `<CardContent />`, is it ok ? :slightly_smiling_face: 
![Screenshot from 2022-08-18 09-51-42](https://user-images.githubusercontent.com/71493888/185343870-6eafa131-9410-4093-bd84-60cd2f8d7b07.png)
![Screenshot from 2022-08-18 10-06-46](https://user-images.githubusercontent.com/71493888/185343888-643901f8-82b4-4ad8-9daa-6d32f2b67124.png)

Reply:
Yes, thanks, I added the missing `<CardContent>` indeed.